sound/soc/sof/ipc4-topology.c

```diff
@@ -1243,6 +1243,16 @@ sof_ipc4_init_base_module_cfg(struct snd_sof_dev *sdev,
 	/* Update the base_config->cpc from the module manifest - max CPC value */
 	sof_ipc4_update_cpc_from_manifest(sdev, swidget->module_info, base_config, false);
 
+	if (!base_config->cpc) {
+		struct sof_ipc4_fw_data *ipc4_data = sdev->private;
+
+		base_config->cpc = ipc4_data->max_core_frequency;
```

Comment:
What is the definition of CPC? Cycles per chunk?

This looks like CPS (cycles per second).

Different but related values.